frontend/javascript/index.js

```diff
@@ -6,13 +6,13 @@ import components from "bridgetownComponents/**/*.{js,jsx,js.rb,css}"
 
 console.info("Bridgetown is loaded!")
 
-function getNextThirdTuesday() {
+function getNextNthTuesday(n) {
```

Comment:
Rather than changing this to `getNextFirstTuesday`, I'm generalizing it to make it easier to change the date again in the future.